compute_tools/src/bin/fast_import.rs

```diff
@@ -217,16 +217,37 @@ pub(crate) async fn main() -> anyhow::Result<()> {
         )
         .instrument(info_span!("postgres")),
     );
+
+    // Create neondb database in the running postgres
     let restore_pg_connstring =
         format!("host=localhost port=5432 user={superuser} dbname=postgres");
```

Comment:
I'd suggest constructing a tokio Config object, something like:
```
let restore_postgres_connstr = Config::new()
    .host("localhost")
    .port(5432)
    .user(superuser)
    .dbname("postgres");

let restore_neondb_connstr = restore_postgres_connstr.dbname("neondb");
```
This avoids the string manipulations. As long as the strings are constants like this, string manipulation works too, but it's error-prone if you have any parts that are configurable or depend on the data.

Reply:
I like both ideas, but I will do it in https://github.com/neondatabase/neon/pull/10407 if you don't mind, it will be hard to rebase already, this will make it even more complicated

Reply:
sure